component/Components.hx

```diff
@@ -3,4 +3,4 @@ package oc.component;
 import oc.component.components.*;
 import haxe.extern.EitherType;
 
-typedef Components = EitherType<ScreenComponent, GpuComponent>;
\ No newline at end of file
+typedef Components = EitherType<ScreenComponent, GpuComponent, ChunkloaderComponen>;
```

Comment:
The EitherType allows only 2 Types, see http://api.haxe.org/haxe/extern/EitherType.html

So this should be:
```haxe
EitherType<ScreenComponent, EitherTypeGpuComponent, ChunkloaderComponent>>
```

Also, it looks like you made a typo, you're missing the `t`.